lmod_proxy/tests/test_edx_grades.py

```diff
@@ -320,6 +320,34 @@ def test_post_grades_approve(self):
         )
         self.assertEqual(data, [])
 
+    def test_max_points(self):
+        """Validate max points arguments"""
+        from lmod_proxy.edx_grades.forms import EdXGradesForm
+        from lmod_proxy.edx_grades.actions import post_grades
```

Comment:
I adapted an existing test that used imports like this. I'll move them all to the top